src/main/java/io/github/terra121/projection/GeographicProjection.java

```diff
@@ -277,4 +277,42 @@ public float azimuth(double x, double y, float angle) throws OutOfProjectionBoun
 	public enum Orientation {
 		none, upright, swapped
 	}
+	public enum Facing {
```

Comment:
Maybe it's just me but I find the name `Facing` kind of confusing, I would find `CardinalDirection` or something like that more explicit. What are your opinions on this ?

Reply:
Also this is quite big, it may be more practical to have it in it's own file to keep `GeographicProjection` clean.

Reply:
You mean the enum its own class or both the enum and method

Reply:
The enum in it's own file, with the method staying in the enum.